File: submissions/sm_112_soumik/week_03/day_3/session_2/average_divisible.js
var limit = 23
var n = 3
var x =0 
var sum=0
for(i =0 ; i< limit ;i++){
    if(i % n == 0){
    sum = sum + i
    x =x + 1
}
}
console.log(x)
console.log(sum/x)

File: submissions/sm_009_chinmay/week_13/day_4/part_1/average_divisible.py
# Print the average of all the numbers from 0 to the given limit that are divisible by the given number (HINT: User range())

limit = int(input('Enter limit: '))
num = int(input('Enter number: '))
sum = 0
cnt = 0

for i in range(limit+1):
    if i % num == 0:
        sum = sum + i
        cnt = cnt + 1

print(sum/cnt)